app/components/course-page/course-stage-step/earned-badge-notice.hbs

```diff
@@ -1,31 +1,27 @@
 {{! @glint-nocheck: not typesafe yet }}
```

Comment:
@ryan-gang when editing files, I'd recommend converting them to typescript as you go - ensures that we aren't introducing new bugs that we could've caught using types. 

(Only exception here would be if the file is huge and the change is minimal)